C/day3.c

```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
+
+int main(){
+    int N; 
+    scanf("%d", &N);
+    if (N % 2 == 1 || (N % 2 == 0 && N <=20 && N >= 6)) { 
+        printf("Weird");
+    } else if (N % 2 == 0 && N <= 5 && N >= 2 || N > 20) {
```

Comment:
Due to operator precedence, the expression `N % 2 == 0 && N <= 5 && N >= 2 || N > 20` is equivalent to `(((N % 2 == 0 && N <= 5) && N >= 2) || N > 20)` (i.e. `&&` has a higher priority than `||`), while the right order should be `(N % 2 == 0 && ((N <= 5 && N >= 2) || N > 20))`, so you need to force a different evaluation order with parenthesis. **In general** this is a problem, since you should print `Not Weird` only if `N` is `even`. In this particular solution, anyway, this problem cannot happen thanks to the previous `if` statement.